### ISSUE TITLE
I can´t open my .epgz file

### ISSUE THREAD

**Issue by Mohamed-Hamid** [evolus/pencil#335]
* Operating system: Ubuntu 14.04
* Pencil version: 3.0.2

I saved my file yesterday. I tried opening it today without a success. It does not give me any error message. I also tried an older version on Windows and it didn't work.

Attached my project file. Please help me as soon as possible as this is critical to my work.

Thank you.

[SmeSpace.tar.gz](https://github.com/evolus/pencil/files/1040912/SmeSpace.tar.gz)

**Comment by dgthanhan** [comment]
Your file was sent.
Please give me more details on the way bitmaps are embeded into your document. We need your help in isolating the cases where saving process generates corrupted file.

**Comment by Mohamed-Hamid** [comment]
Thank you @dgthanhan and sorry for the late reply. Actually I was copying images from Chrome and pasting them directly in pencil without saving them locally. I hope this piece of information helps you find the root cause.

**Comment by dgthanhan** [comment]
@Mohamed-Hamid The usecase where image was copied as binary from image editing software, screenshot or browser is indeed an important usecase that we would like Pencil to suppor well. We definitely don't want users to spend and extra step to store that binary data as a file before inserting.
The process itself may not have problem but the size of it may be the root cause. Is your image a large one?

**Comment by Mohamed-Hamid** [comment]
@dgthanhan Sure it's an important feature that I was delighted when I tried it and found it working.
Images were not large. Something between 100~300KB. Please let me know if I could help with any other information.

**Comment by dgthanhan** [comment]
V3.0.3 has additonal check to avoid unexpecture process shutdown when saving is in-progress. Could you please try it in your normal/daily uses to see if you encounter the problem again.

**Comment by Mohamed-Hamid** [comment]
@dgthanhan Great. I can recall that I shutdown my machine a minute after saving. A shutdown *while* saving? maybe. Pencil does not provide any feedback that a saving is in progress so I assumed that it finished fast. 
Sure I will get back if it happened again. Thanks for your help.

**Comment by dgthanhan** [comment]
No, I'm talking about the process shutdown which means the termination of the Pencil process in general, not necessarily mean the machine shutdown which is a kind of an extreme case.

**Comment by Mohamed-Hamid** [comment]
I understand that. I'm just stating that maybe it happened because of a deliberate termination of the process while saving.
